sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_models.py

```diff
@@ -236,17 +231,15 @@ class TextSource(object):
     """Endpoint where the custom voice was deployed"""
 
     def __init__(
-            self,
-            text: str,
-            *,
-            source_locale: Optional[str] = None,
-            voice_gender: Optional[Union[str, 'Gender']] = None,
-            voice_name: Optional[str] = None,
-            play_source_cache_id: Optional[str] = None,
-            custom_voice_endpoint_id: Optional[str] = None,
-            **kwargs
+        self,
+        text: str,
+        *,
+        source_locale: Optional[str] = None,
+        voice_gender: Optional[Union[str, 'Gender']] = None,
+        voice_name: Optional[str] = None,
+        play_source_cache_id: Optional[str] = None,
+        custom_voice_endpoint_id: Optional[str] = None
```

Comment:
**kwargs?

Reply:
We don't want kwargs in a model constructor. There's no pass-through and it allows the possibility of accidentally using a typo in a parameter name and it failing silently. I've removed it from non-GA models. For models that are already GA, I've left it in place for now.